### PR TITLE
QX 2611 pdf printing from viewer

### PR DESCRIPTION
**Pull request by jmillarMC**
Updating to allow a flag to be passed to allow the pdf to be printed from the viewer

### CODE REVIEW

File: web/pdf_print_service.js [hunk 1]
```diff
@@ -16,6 +16,7 @@
 import { CSS_UNITS, NullL10n } from './ui_utils';
 import { PDFPrintServiceFactory, PDFViewerApplication } from './app';
 import { URL } from 'pdfjs-lib';
+import {MCOptions} from "./mc_options";
```
Comment (jjbradshaw) [inline web/pdf_print_service.js:19]: Should be single quoted for consistency.

File: web/viewer.css [hunk 2]
```diff
@@ -1944,7 +1940,7 @@ html[dir='rtl'] #documentPropertiesOverlay .row > * {
 }
 
 @media print {
-  body {
+  .disablePrint {
```
Comment (jjbradshaw) [inline web/viewer.css:1943]: Can you explain the purpose of this? I'm having a hard time understanding it. My understanding is that we're hiding the PDF and the toolbar if we're in print preview mode. Not really sure why we would do that.

File: web/viewer.js [hunk 3]
```diff
@@ -200,6 +213,8 @@ function webViewerLoad() {
     if (typeof PDFJSDev !== 'undefined' && PDFJSDev.test('CHROME')) {
       pdfjsWebAppOptions.AppOptions.set('defaultUrl', defaultUrl);
     }
+    setMCOptions();
```
Comment (jjbradshaw) [inline web/viewer.js:216]: Do we really benefit by having two consecutive methods, one to set the option, and one to review the option and act accordingly? Feels like we can condense this down into a single method and we would be able to get rid of the MCOptions entirely. Can we not simply check the url param and act on the document immediately?

**Review by jjbradshaw — COMMENTED** [review]
Can we update this PR with a meaningful description?

File: web/viewer.css (hunk 2 above)
Comment (jmillarMC) [reply web/viewer.css:1943]: We are making it a class and moving the application of it to be more specific. Without doing that we ran into a problem with ctrl+P working.

File: web/viewer.js (hunk 3 above)
Comment (jmillarMC) [reply web/viewer.js:216]: The idea we had is setting up the pattern for the future for passing things and storing them in pdf.js. I could see how that could be seen as to much "future proofing"

Comment (jjbradshaw) [reply web/viewer.js:216]: Makes sense. I would consider it over engineering but it's already there so I'm not concerned about it. This is only the second change I'm aware of to this library in the last two years and I don't think anything is on the roadmap for future enhancements.

**Review by robkacic — COMMENTED** [review]
LGTM!

**Pull request merged**